test/parallel/test-crypto-x509.js

```diff
@@ -359,3 +359,37 @@ UcXd/5qu2GhokrKU2cPttU+XAN2Om6a0
 
   assert.strictEqual(cert.checkIssued(cert), false);
 }
+
+{
+  // Test X509Certificate extension methods
+
+  const certPemWithExtensions = `
```

Comment:
maybe add this file to fixtures instead of inline? It's not formatted and indentet properly to be a valid PEM string.